gatsby/src/pages/drugs/{BnfDrug.slug}/medicinal-forms.tsx

```diff
@@ -54,27 +89,100 @@ const MedicinalFormsPage: FC<MedicinalFormsPageProps> = ({
 			id: slug,
 			title: form,
 		}))}
+		asideContent={asideInfo}
+		headerCta={
+			<Link to={`/drugs/${slug}/`}>
+				View monograph<span className="visually-hidden"> for {title}</span>
```

Comment:
There are some drugs where the title includes HTML e.g. Anti Rh₀(D) ... (with a subscript 0) so we'd need to use `dangerouslySetInnerHTML` here